From e80fca2d5121c86d755cafaf3b0009ca1c5d3727 Mon Sep 17 00:00:00 2001
From: Aditya Singh <v-aditysing@microsoft.com>
Date: Wed, 1 Jul 2026 07:29:32 +0000
Subject: [PATCH] update-format-function-checks-in-panic.go

In newer Go versions, format-function checks are stricter.
Fprintf expects a format string + optional args. A dynamic
string can trigger vet/compile diagnostics in some toolchain setups.
---
 panic.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/panic.go b/panic.go
index 6f60964..b7f8be1 100644
--- a/panic.go
+++ b/panic.go
@@ -36,7 +36,7 @@ func panicHandler(logF *os.File) panicwrap.HandlerFunc {
 	return func(m string) {
 		// Write away just output this thing on stderr so that it gets
 		// shown in case anything below fails.
-		fmt.Fprintf(os.Stderr, fmt.Sprintf("%s\n", m))
+		fmt.Fprintf(os.Stderr, "%s\n", m)
 
 		if err := packer.CheckpointReporter.ReportPanic(m); err != nil {
 			fmt.Fprintf(os.Stderr, "Failed to report panic. This is safe to ignore: %s", err)
-- 
2.45.4

